doc/jinja-filters.md

```diff
@@ -0,0 +1,248 @@
+# Jinja Filters Used by Schemas
+
+###### Date 14/12/2018
+
+## format_number
+Returns the given number formatted for a specific locale. 
+Uses Babel's number.py and format_decimal.
+
+##### Parameters: 
+- value: A single numeric value.
+
+##### Context:
+Most commonly used to display a previous answer's numeric value in a following question.
+<br />
+`Of the <em>{{answers['total-number-employees']|format_number}}</em> total employees employed on 14 December 2018, how many male and female employees worked the following hours?`
+<br />
+
+
+## format_currency
+Returns a formatted currency value. Uses Babel's number.py and format_currency.
+
+##### Parameters: 
+- value: A single numeric value.
+- currency: An optional currency in format 'GBP' (default), 'EUR' etc.
+
+##### Context:
+Most commonly used to display a previous answer's currency value in a following question.
+<br />
+`Of the <em>{{format_currency(answers['total-retail-turnover-answer'])}}</em> total retail turnover, what was the value of internet sales?`
+<br />
+
+
+## format_address_list
+This function format_address_list accepts two addresses, a user submitted address and a metadata address. 
+If all the items in the user entered address list are 'Undefined' (nothing in answer_store) then we retrieve the metadata.
+The address list is returned with break points between each address line.
+
+##### Parameters: 
+- user_entered_address: List of address values from answers
+- metadata_address: List of address values from metadata
+
+##### Context:
+Used to display a full address for use in a description.
+<br />
+`"description": "{{ format_address_list ([answers['address-line1'], answers['address-line2'], answers['locality'], answers['town-name'], answers['postcode']], [metadata['address_line1'], metadata['address_line2'], metadata['locality'], metadata['town_name'], metadata['postcode']]) }}"`
+<br />
+
+
+## get_current_date
+Returns the current Date.
+
+##### Parameters: 
+- N/A
+
+##### Context:
+No real current usage but a single question in Census I believe only for test purposes.
+<br />
+`The number of visitors staying in the household on {{ get_current_date() }}`
+<br />
+
+
+## format_date
+Formats a date string, returning "d MMMM YYYY", "MMMM YYYY" or "YYYY" depending on value passed. 
```

Comment:
I think this needs to be clearer. What values can be passed, and what will the output be for those values?